### DISCUSSION TITLE
[react-native-video] contains node_modules directory

### DISCUSSION THREAD

**Discussion by bitboxer** [DefinitelyTyped/DefinitelyTyped#56788] [Issues with a @types package]
The package for [@types/react-native-video](https://www.npmjs.com/package/@types/react-native-video) contains a `node_modules` directory that leads to lots of `duplicate types` warnings.

[a comment by an automated account is not shown]

**Reply by samarmohan** [reply]
I don't think it's there anymore.
![image](https://user-images.githubusercontent.com/71091489/138940924-049d682f-6121-437e-bd9d-1a0642293a9f.png)

**Reply by bitboxer** [reply]
Sorry for the late reply, the notification got lost in my spam folder 😭 .

The folder is not present on github, but is in the package that is on npmjs. If I install the package I see this:

<img width="141" alt="Screen Shot 2021-11-01 at 12 38 42" src="https://user-images.githubusercontent.com/56195/139665962-99fe444d-79dd-48fc-b660-2b6f5f36192f.png">
